src/components/antdComponents/antdform/AntdForm.tsx

```diff
@@ -0,0 +1,59 @@
+// Imports
+import { useAuth0 } from '@auth0/auth0-react';
+import { Form, Button } from 'antd';
+// Local Imports
+import AutoCompleteForm from './AutoCompleteForm';
+
+interface AntdFormProps {
+  data: {
+    name: string;
+    // eslint-disable-next-line
+    rules: any[];
+    content: JSX.Element;
+  }[];
+}
+
+const AntdForm = (props: AntdFormProps) => {
+
+  const { user } = useAuth0();
+
+  // Inside this function we call api service. Need to check if the SYMBL exists
+  const onSubmit = (values) => {
+    console.log('Success:', values, user?.sub);
```

Comment:
perfect thank you, will add this service in tomorrow, can pair program it if you like